docs/docs/20-usage/80-badges.md

```diff
@@ -15,4 +15,11 @@ The status badge displays the status for the latest build to your default branch
 +<scheme>://<hostname>/api/badges/<repo-id>/status.svg?branch=<branch>
 ```
 
-Please note status badges do not include pull request results, since the status of a pull request does not provide an accurate representation of your repository state.
+The badge represents the state of the last push event. If you'd like to respect other or further events, you can add the `events` query parameter:
+
+```diff
+-<scheme>://<hostname>/api/badges/<repo-id>/status.svg
++<scheme>://<hostname>/api/badges/<repo-id>/status.svg?events=manual,cron
+```
+
+Please note status badges should not include pull request results, since the status of a pull request does not provide an accurate representation of your repository state.
```

Comment:
Can we move this line over l. 18 and change to something like this
```suggestion
By default status badges do not include pull request results, since the status of a pull request does not provide an accurate representation of your repository state.
```